src/libs/site.ts

```diff
@@ -11,7 +11,7 @@ const siteData = {
     title: "CWTS 基督工人神學院",
     shortTitle: "基督工人神學院",
     home: "/",
-    coverVideoTitle: "基督工人神學院 CWTS Merry Christmas 2023",
+    coverVideoTitle: "神學生見證 - 神學學習如何提升和改變我",
```

Comment:
There is an English translation above. Please update it as well.

Reply:
Youtube video English title updated